spaces/domains.json

```
@@ -21,6 +21,7 @@
   "vote.mcafeemoonshot.com": "mcafeemoonshot.eth",
   "vote.fbomb.finance": "fbomb.eth",
   "vote.evermoon.io": "evermoon.eth",
+  "ethisfunny.eth": "agld-dao.eth",
```

You should create a subdomain for your space.
```suggestion
  "vote.ethisfunny.eth": "agld-dao.eth",
```